### DISCUSSION TITLE
DotPlot : Plot markers per Cluster - Gene Expression & n° of Cells expressing the marker

### DISCUSSION THREAD

**Discussion by matteozoia4** [GreenleafLab/ArchR#1749] [Questions / Documentation]
Dear all,

I would like to retrieve for several gene-markers their expression and % of cells expressing it, this per each cluster.
I would need a DotPlot-like figure, how could I approach this representation?

Should I use GeneScore or GeneExpression matrix?
And more importantly, is it possible to apply DotPlot format?

Many thanks for your advices,

MZ

**Comment by rcorces** [comment]
ArchR does not provide infrastructure for this. Certainly GeneScoreMatrix cannot be accurately used for this. You will have to obtain whatever data you are interested in from ArchR, likely using `getMatrixFromProject()` or `exportGroupSE()` and perform your manipulations and plotting on your own

**Comment by matteozoia4** [comment]
Many thanks!


Matteo Zoia, Ph.D. Student
Gene Regulation & Cardiac Development
Osterwalder Lab
Department for BioMedical Research (DBMR)
University of Bern
Murtenstrasse 28, 369
CH - 3008 Bern
SWITZERLAND
________________________________
Da: Ryan Corces ***@***.***>
Inviato: lunedì, 21. novembre 2022 15:30:22
A: GreenleafLab/ArchR
Cc: Zoia, Matteo (DBMR); Author
Oggetto: Re: [GreenleafLab/ArchR] DotPlot : Plot markers per Cluster - Gene Expression & n° of Cells expressing the marker (Discussion #1749)


ArchR does not provide infrastructure for this. Certainly GeneScoreMatrix cannot be accurately used for this. You will have to obtain whatever data you are interested in from ArchR, likely using getMatrixFromProject() or exportGroupSE() and perform your manipulations and plotting on your own

**Reply by Zepeng-Mu** [reply]
https://github.com/GreenleafLab/ArchR/discussions/882
